reflex/utils/format.py

```diff
@@ -253,24 +253,21 @@ def format_cond(
     # Use Python truthiness.
     cond = f"isTrue({cond})"
 
+    def create_cond_part_var(cond_part):
+        return Var.create_safe(cond_part, _var_is_string=type(cond_part) is str)
+
     # Format prop conds.
     if is_prop:
         if not isinstance(true_value, Var):
-            true_value = Var.create_safe(
-                true_value,
-                _var_is_string=type(true_value) is str,
-            )
+            true_value = create_cond_part_var(true_value)
```

Comment:
We can do this unconditionally (don't have to do is instance checks) since create_safe will just return the Var itself it its not already a var